testbed/coav-sim.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
+
+# Supported Autopilots
+AP_PX4=0
+AP_APM=1
+
+# ArduCopter Variables
+APM_CMD="arducopter"
+APM_TCP_PORT_1=5760
+APM_TCP_PORT_2=5762
+
+# PX4 Variables
+PX4_DIR=${PX4_DIR:-"~/px4/Firmware"}
```

Comment:
Would be nice to have the same "prefix" capabilities for ardupilot, I always have to edit the script to provide the full path